openedx/core/djangoapps/theming/views.py

```diff
@@ -0,0 +1,104 @@
+"""
+Views file for the Darklang Django App
```

Comment:
Old comment.  Maybe do a search for `lang` in this folder to make sure we didn't miss anything?